client.js

```diff
@@ -101,13 +101,11 @@ var System = {
     get: get,
     has: has,
     import: function(name) {
-        return new Promise(function(resolve, reject) {
-            var normalizedName = normalizeName(name, []);
-            var mod = get(normalizedName);
```

Comment:
I see the problem here, but removing the around promise is not the solution since that promise protects issues with normalizeName and get. 

The problem here is really that resolve is never called when load is invoked. The solution should be to propagate resolve and reject a then() when calling load().